examples/shared_kv_storage/kv_storage/remote_lmdb/test_lmdb_app.py

```diff
@@ -0,0 +1,190 @@
+                         "get_value Retrieved incorrect value")
+
+    def test_remove(self):
+        remove_result = self.proxy.remove(self.table, self.key)
+        logger.info(remove_result)
+        self.assertTrue(remove_result, "remove Failed")
+
+    def test_get_none(self):
+        get_none_result = self.proxy.get(self.table, self.key)
+        logger.info(get_none_result)
+        self.assertIsNone(get_none_result,
+                          "get_none Did not return None for empty value")
+
+    def test_remove2(self):
+        remove_result2 = self.proxy.remove(self.table, self.key2)
+        logger.info(remove_result2)
+        self.assertTrue(remove_result2, "remove Failed")
+
+    def test_lookup_none(self):
+        lookup_none_result = self.proxy.lookup(self.table)
+        logger.info(lookup_none_result)
+        self.assertEqual(lookup_none_result, [],
+                         "lookup_none Did not return empty array")
+
+# -----------------------------------------------------------------
+
+
+def local_main():
+    result = unittest.TestResult()
+    result.startTestRun()
+    test = TestRemoteLMDB()
+    test.test_set()
+    test.test_set2()
+    test.test_lookup()
+    test.test_get_value()
+    test.test_get_value2()
+    test.test_remove()
+    test.test_get_none()
+    test.test_remove2()
+    test.test_lookup_none()
+    result.stopTestRun()
+    if result.wasSuccessful():
+        logger.info("All tests passed!")
+    else:
+        logger.info("Some tests failed.")
+    exit(0)
+
+# -----------------------------------------------------------------
+
+
+def parse_command_line(config, args):
+
+    global bind_uri
+
+    parser = argparse.ArgumentParser()
+
+    parser.add_argument('--logfile',
+                        help='Name of the log file, __screen__ for standard output',
+                        type=str)
+    parser.add_argument('--loglevel',
+                        help='Logging level',
+                        type=str)
+    parser.add_argument('--bind_uri',
+                        help='URI to send requests to',
+                        type=str,
+                        default='http://localhost:9091')
+
+    options = parser.parse_args(args)
+
+    if config.get('Logging') is None:
+        config['Logging'] = {
+            'LogFile': '__screen__',
+            'LogLevel': 'INFO'
+        }
+    if options.logfile:
+        config['Logging']['LogFile'] = options.logfile
+    if options.loglevel:
+        config['Logging']['LogLevel'] = options.loglevel.upper()
+    if options.bind_uri:
+        bind_uri = options.bind_uri
+
+# -----------------------------------------------------------------
+# -----------------------------------------------------------------
+
+
+def main(args=None):
+    import config.config as pconfig
+    import utility.logger as plogger
+
+    # Parse out the configuration file first
+    conffiles = ['tcs_config.toml']
+    confpaths = [".", TCFHOME + "/config"]
```

Comment:
Since this is only one conffile, not multiple, change `conffiles` to `conffile`.
Same with `confpaths`

Reply:
conffiles and confpaths are of type list that why there are named like this.